src/formula/variant.cpp

```diff
@@ -841,97 +634,49 @@ bool variant::operator>(const variant& v) const
 
 variant variant::list_elements_add(const variant& v) const
 {
-	must_be(TYPE_LIST);
-	v.must_be(TYPE_LIST);
-
-	if( num_elements() != v.num_elements() )
-		throw type_error("Operator '.+' requires two lists of the same length");
```

Comment:
So I guess we've lost this nice error message? At least, the mention of the specific operator in it.